tests/network/setup_multimachine.pm

```diff
@@ -36,10 +36,16 @@ sub run {
     # Configure the internal network an  try it
     if ($hostname =~ /server|master/) {
         setup_static_mm_network('10.0.2.101/24');
-        #if server running opensuse.
+        #if server running openSUSE.
         if (is_opensuse) {
-            disable_and_stop_service('NetworkManager', ignore_failure => 1);
-            assert_script_run 'systemctl start  wicked';
+            assert_script_run 'systemctl restart  wicked';
+        }
+        #if server runing on SLED
+        my $base_product = get_var('SLE_PRODUCT');
+        if ($base_product eq "sled") {
```

Comment:
This would likely lead to perl warnings about comparison against undefined variables. Better use

```suggestion
        if (check_var('SLE_PRODUCT', 'sled')) {
```

but in general I doubt it's a save approach to rely on a specific product brand to derive if wicked or NetworkManager is used.

Reply:
Ok, sounds good. 

I think maybe a new ticket to verify this, a better approach to this situation with product brand and network.